report_thesis/src/index.tex

```diff
@@ -3,7 +3,11 @@
 \newpage
 
 \begin{abstract}
-    Abstract
+This thesis advances the analysis of \gls{libs} data for predicting major oxide compositions in geological samples.
+By integrating machine learning techniques and ensemble regression models, the study addresses challenges like high dimensionality, non-linearity, multicollinearity, and limited data availability.
```

Comment:
```suggestion
By integrating machine learning techniques and ensemble regression models, the study addresses challenges like high dimensionality, multicollinearity, and limited data availability.
```

Non linearity is definitely an issue, but we havent highlighted it as a challenge.